Specifications/Language/3_Expressions/ComparativeExpressions.md

```diff
@@ -22,7 +22,7 @@ For instance, the following comparison evaluates to `false` due to rounding erro
 >```
 >for some globally declared function `Bar`. The first line defines a new anonymous function that takes an argument `x` and invokes a function `Bar` with it and assigns it to the variable `f1`. The second line assigns the function `Bar` to `f2`. Since invoking `f1` and invoking `f2` will do the same thing, it should be possible to replace those with each other without changing the behavior of the program. This wouldn't be the case if the equality comparison for functions was supported and `f1 == f2` evaluates to `false`. If conversely `f1 == f2` were to evaluate to `true`, then this leads to the question of determining whether two callable will have the same side effects and evaluate to the same value for all inputs, which is not possible to reliably determine. Therefore, if we would like to be able to replace `f1` with `f2`, we can't allow equality comparisons for callables.  
 
-# Quantitative Comparison
+## Quantitative Comparison
```

Comment:
This is also not the correct hierarchy. If we need exactly one H1 header, then adding a header 
# Comparative Expressions
and making both of the subheaders H2 would make sense.